app/models/reservation.rb

```diff
@@ -0,0 +1,12 @@
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :event
+  belongs_to :skill
+  belongs_to :reservation_status
+  has_many :driven_pairs, class_name: 'Pair', foreign_key: 'person_1'
+  has_many :navigated_pairs, class_name: 'Pair', foreign_key: 'person_2'
+
+  def pairs
+    driven_pairs + navigated_pairs
```

Comment:
It makes sense that driver and navigator are important and helpful to establish for a pair. I could see how it would be helpful to know how many pairs someone has driven or navigated, both for the person and the org.